docs/guide/installation.md

```diff
@@ -55,6 +55,20 @@ echo "https://alpine.fury.io/rgst-io/" >> /etc/apk/repositories
 curl https://alpine.fury.io/rgst-io/rgst-io@fury.io-946e9786.rsa.pub | sudo tee /etc/apk/keys/'rgst-io@fury.io-946e9786.rsa.pub' >/dev/null
 ```
 
+### Github Action
+
+Quickstart:
+
+```yaml
+  - name: Install Stencil
+    uses: rgst-io/stencil-action@latest
+    with:
+      github-token: ${{ github.token }}
```

Comment:
why are you passing a github token here?  do you need to create a custom PAT for this to work?

Reply:
It is required to create the octo client in a Github action, apparently. It's only actually used when `version` is `latest`.

That's not a PAT though, just the built-in token thankfully